Standards/scs-0004-v1-achieving-certification.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Regulations for achieving SCS-compatible certification
+type: Procedural
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The Sovereign Cloud Stack (SCS) issues certificates with various scopes, among them _SCS-compatible IaaS_ (infrastructure as a service) and _SCS-compatible KaaS_ (Kubernetes as a service).
+
+This document details how a cloud service provider (henceforth also called operator) can attain such a certificate for one of their clouds.
+
+## Motivation
+
+As operator, I want to obtain a certificate with the scope SCS-compatible IaaS or SCS-compatible KaaS.
+
+## Regulations
+
+1. Each certificate issued pertains to a given cloud, a given scope, and a given version of that scope with a fixed expiry date. The certificate is only valid for that cloud and for the time frame that ends on that expiry date.
```

Comment:
As each certificate version has an expiry date and each certificate has an expiry date, would it makes sense, to add a sentence how these dates interrelate? Do we support certificate expiry dates that are later in time than the expiry date of the corresponding certificate version? Or are certificates maximum valid until the expiry date of certificates' version?

Reply:
Good point. I always assumed it was self-evident that the certificate expires at the same date as the version of the scope; however, it could also expire earlier. So it does make sense to make this explicit! I think we agreed that the certificate should not expire earlier, right?

Reply:
On second thought, the sentence already states precisely that:
> The certificate is valid ... for the time frame that ends on that expiry date.

Reply:
Marking this as resolved because @fdobrovolny raised a similar question. Let's address it there.